newswires/client/src/urlState.ts

```diff
@@ -74,8 +97,37 @@ export const configToUrl = (config: Config): string => {
 	}
 };
 
+const processDateMathRange = (config: Query, useDateTimeValue: boolean) => {
+	if (useDateTimeValue) {
+		if (config.dateRange) {
```

Comment:
could we add a comment along the lines of "The API takes fully ISO timestamp strings, so if we're preparing the params for an API call, we need to convert any relative time info to ISO timestamps"

(assuming I'm understanding the logic here correctly!)